src/resolve/adapters/digitalExperienceSourceAdapter.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { dirname, join, sep } from 'path';
+import { META_XML_SUFFIX } from '../../common';
+import { SourceComponent } from '..';
+import { MetadataXml } from '../types';
+import { baseName, parentName } from '../../utils';
+import { SourcePath } from '../../common';
+import { BundleSourceAdapter } from './bundleSourceAdapter';
+
+/**
+ * Source Adapter for DigitalExperience metadata types. This metadata type is a bundled type of the format
+ *
+ * __Example Structure__:
+ *
+ *```text
+ * site/
+ * ├── foos/
+ * |   ├── sfdc_cms__appPage/
+ * |   |   ├── mainAppPage/
+ * |   |   |  ├── _meta.json
+ * |   |   |  ├── content.json
+ * |   ├── sfdc_cms__view/
+ * |   |   ├── view1/
+ * |   |   |  ├── _meta.json
+ * |   |   |  ├── content.json
+ * |   |   |  ├── fr.json
+ * |   |   |  ├── en.json
+ * |   |   ├── view2/
+ * |   |   |  ├── _meta.json
+ * |   |   |  ├── content.json
+ * |   |   |  ├── ar.json
+ * |   ├── foos.digitalExperience-meta.xml
+ * content/
+ * ├── bars/
+ * |   ├── bars.digitalExperience-meta.xml
+ * ```
+ *
+ * In the above structure the metadata xml file ending with "digitalExperience-meta.xml" belongs to DigitalExperienceBundle MD type.
+ * The "_meta.json" files are child metadata files of DigitalExperienceBundle belonging to DigitalExperience MD type. The rest of the files in the
+ * corresponding folder are the contents to the DigitalExperience metadata. So, incase of DigitalExperience the metadata file is a JSON file
+ * and not an XML file
+ */
+export class DigitalExperienceSourceAdapter extends BundleSourceAdapter {
```

Comment:
@shetzel just noticed https://github.com/forcedotcom/source-deploy-retrieve/blob/aa79e2642a7aa35b05cb71c9f4cc0b0c21c21123/src/resolve/adapters/index.ts#L7 Should I add this adapter to index.ts?

Reply:
@Kaaviyan - yes, please add it.  Good catch!